src/OSmOSE/Spectrogram.py

```diff
@@ -126,7 +126,7 @@ def __init__(
             analysis_sheet["overlap"][0] if "overlap" in analysis_sheet else None
         )
         self.__colormap: str = (
-            analysis_sheet["colormap"][0] if "colormap" in analysis_sheet else None
+            analysis_sheet["colormap"][0] if "colormap" in analysis_sheet else "viridis"
         )
```

Comment:
i thougt we could start entering no None values here as default ..

Reply:
Indeed, if some values can have default values we can set them here.